src/test/java/uk/gov/hmcts/ccd/domain/service/common/TestBuildersUtil.java

```diff
@@ -375,7 +380,7 @@ private CaseTypeBuilder() {
             caseType.setJurisdiction(new Jurisdiction());
         }
 
-        public static CaseTypeBuilder aCaseType() {
+        public static CaseTypeBuilder anCaseType() {
```

Comment:
What's the motivation behind that change? the name seemed better before.

Reply:
Codacy complains there cannot be a single letter word at the start of method name.

Reply:
Well, that's a pain :D Should we switch to `newCaseType()` or `caseType()` then instead, to try and keep it grammatically correct at least?

Reply:
Sure done